pkg/kubectl/cmd/set/set_selector.go

```diff
@@ -133,10 +134,14 @@ func (o *SelectorOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args [
 
 // Validate basic inputs
 func (o *SelectorOptions) Validate() error {
+	errors := []error{}
```

Comment:
I referred to https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/set/set_image.go#L157 .
Validate() in set_image.go needs to be updated too ?